From e926f6475d2f7063f8190076a0dc9ff7ecb227c8 Mon Sep 17 00:00:00 2001
From: Thomas Hindoe Paaboel Andersen <phomes@gmail.com>
Date: Thu, 11 Sep 2014 21:29:59 +0200
Subject: [PATCH] firstboot: silence a warning

No change in behavoir as the fallthrough from ARG_COPY had already
set arg_copy_locale to true.

Found with coverity. Fixes: CID#1237622
---
 src/firstboot/firstboot.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/firstboot/firstboot.c b/src/firstboot/firstboot.c
index fd73adbac8..215c059ee2 100644
--- a/src/firstboot/firstboot.c
+++ b/src/firstboot/firstboot.c
@@ -856,6 +856,7 @@ static int parse_argv(int argc, char *argv[]) {
 
                 case ARG_COPY:
                         arg_copy_locale = arg_copy_timezone = arg_copy_root_password = true;
+                        break;
 
                 case ARG_COPY_LOCALE:
                         arg_copy_locale = true;
